spec/02-integration/04-admin_api/07-upstreams_routes_spec.lua

```diff
@@ -663,6 +663,29 @@ describe("Admin API: #" .. strategy, function()
           assert.same({ data = {}, next = ngx.null }, json)
         end)
       end)
+
+      describe("#regression", function()
```

Comment:
This test suite uses the iterator `each_strategy`, which only iterates over DB strategies, so effectively this is not testing against db-less.

Reply:
couls use all_strategies（）

Reply:
changed to use all_strategies